tests/e2e/conftest.py

```diff
@@ -21,6 +21,8 @@ def cluster(kind_cluster) -> Generator[dict, None, None]:
     kind_cluster.load_docker_image(webhook_image)
 
     logging.info("Deploying CertManager")
+    kubectl("delete", "daemonset", "-n" "kube-system", "kindnet")
+    kubectl("apply", "-f", "https://docs.projectcalico.org/v3.8/manifests/calico.yaml")
```

Comment:
Please update this calico deployment to point to 3.16

Reply:
Done, I updated on installation script not on tests :(